salt/utils/mount.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+'''
+Common functions for managing mounts
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+import os
+import yaml
+
+# Import Salt libs
+import salt.utils  # Can be removed once is_true is moved
+import salt.utils.files
+import salt.utils.versions
+
+from salt.utils.yamldumper import SafeOrderedDumper
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    '''
+    Confine this module to Debian based distros
```

Comment:
This is minor, but this docstring is confusing since it's saying the module is confined to Debian, but then it just returns `True` and doesn't do any checking.